src/hooks/makeMutation.ts

```diff
@@ -47,39 +52,43 @@ export function useMutation<TData, TVariables>(
   const [mutateFn, result] = useBaseMutation(mutation, {
     ...opts,
     onCompleted: data => {
-      handleNestedMutationErrors({
-        data,
-        intl,
-        notify,
-      });
+      if (!disableErrorHandling) {
+        handleNestedMutationErrors({
+          data,
+          intl,
+          notify,
+        });
+      }
 
       if (onCompleted) {
         onCompleted(data);
       }
     },
     onError: (err: ApolloError) => {
-      if (err.graphQLErrors) {
-        if (hasError(err, GqlErrors.ReadOnlyException)) {
-          notify({
-            status: "error",
-            text: intl.formatMessage(commonMessages.readOnly),
-          });
-        } else if (err.graphQLErrors.some(isJwtError)) {
-          user.logout();
-          notify({
-            status: "error",
-            text: intl.formatMessage(commonMessages.sessionExpired),
-          });
-        } else if (!hasError(err, GqlErrors.LimitReachedException)) {
-          err.graphQLErrors.map(graphQLError => {
+      if (!disableErrorHandling) {
```

Comment:
This will disable callback passed in `onError` (line 94). I don't think we should disable it, we might want side effects when handling errors manually